manager/src/test/java/com/redhat/service/smartevents/manager/api/user/BridgesAPITest.java

```diff
@@ -331,6 +334,23 @@ public void testDeleteBridgeWithActiveProcessors() {
         TestUtils.deleteBridge(bridgeResponse.getId()).then().statusCode(400);
     }
 
+    @Test
+    @TestSecurity(user = DEFAULT_CUSTOMER_ID)
+    public void testDeleteBridgeWithActiveErrorHandler() {
+        Action errorHandler = createWebhookAction();
+        BridgeResponse bridgeResponse = TestUtils.createBridge(new BridgeRequest(DEFAULT_BRIDGE_NAME, errorHandler)).as(BridgeResponse.class);
+        TestUtils.updateBridge(
+                new BridgeDTO(bridgeResponse.getId(),
+                        bridgeResponse.getName(),
+                        bridgeResponse.getEndpoint(),
+                        DEFAULT_CUSTOMER_ID,
+                        DEFAULT_USER_NAME,
+                        READY,
+                        new KafkaConnectionDTO()));
+
+        TestUtils.deleteBridge(bridgeResponse.getId()).then().statusCode(202);
+    }
```

Comment:
Is there also an error handler state that should cause bridge deletion to fail? if yes, should we cover it by test?

Reply:
New test added.